src/app/add-item/add-item.component.ts

```diff
@@ -53,14 +51,16 @@ export class AddItemComponent implements OnInit {
   }
 
   private formNewActionItem(): ActionItem {
-    const projectNameValue = this.dialogForm.get('project').value;
-    const dueDateValue = this.dialogForm.get('dueDate').value;
+    const projectNameValue = this.dialogForm.get('project')!.value;
+    const dueDateValue = this.dialogForm.get('dueDate')!.value;
+    const itemId = parseInt(this.tasksService.getLastItemId()) + 1;
```

Comment:
**SUGGESTION:** As in the end usage this `id` will be generated on the server then it would be better to assign it in the service.